metrics/metrics-api/src/main/java/org/apache/james/metrics/api/Metric.java

```diff
@@ -35,4 +35,15 @@
      * Implementation might be doing strict validation (throwing on negative value) or lenient (log and sanitize to 0)
      */
     long getCount();
+
+    /**
+     * Moving average.
+     *
+     * Period of the moving average is implementation defined.
+     *
+     * Default to count (naive implementation with period starting at boot time)
+     */
+    default double movingAverage() {
```

Comment:
I'm starting to think it's not an average at all, it's a windowed counter?

Reply:
I would find a windowed counter easier to mentally represent than a moving average hopefully

Reply:
I wonder if meter allow to reach that goal however.